### PR TITLE
Localisation Crusade - 4th Chapter

### PR DESCRIPTION
**Pull request by DukeJeans**
German and French translation for mod. Improvements to current English localisations.
__________________________
**Changelog:**
>- English improved in 64 lines of localisation.
>- French and German translations added for 3070 lines of localisation.

### CODE REVIEW

File: localisation/wc_religions.csv [hunk 1]
```diff
@@ -163,7 +163,7 @@ forsaken_cult_DESC;This is a group of forsaken considered heretical by most. The
 THE_SHADOW;the Shadow;;;;;;;;;;;;;x;
 void_group;Void;;;;;;;;;;;;;x;
 old_ways;Old Way;;;;;;;;;;;;;x;
-old_ways_DESC;The old ways are not widespread, but have existed on the fringes of Gilnean society. The religious leaders of what was in Gilneas referred to as the "old ways" eventually became "harvest-witches". Those who used their nature powers to augment Gilneas' agricultural output during and following its period of industrialization.;;;;;;;;;;;;;x;
+old_ways_DESC;"The old ways are not widespread, but have existed on the fringes of Gilnean society. The religious leaders of what was in Gilneas referred to as the ""old ways"" eventually became ""harvest-witches"". Those who used their nature powers to augment Gilneas' agricultural output during and following its period of industrialization.";;;;;;;;;;;;;x;
```
Comment (MemeWizard) [inline localisation/wc_religions.csv]: May I ask what quotation marks do in the religion descriptions?

Comment (arithon) [reply localisation/wc_religions.csv]: I think they appeared when someone opened and saved the file with excel... we got double "" as well... Changing it now.

Comment (arithon) [reply localisation/wc_religions.csv]: Someone should check line 189. The French line there seems to be a description for another thing in english...

Comment (DukeJeans) [reply localisation/wc_religions.csv]: Some sort of problem with my Surface's OOC involving the CF character. I'll dig the problem out coming up.

Comment (DukeJeans) [reply localisation/wc_religions.csv]: Fixed in ccf7a7a

Comment (MemeWizard) [reply localisation/wc_religions.csv]: I'm talking about it
Безымянный

Comment (arithon) [reply localisation/wc_religions.csv]: Yes, I know. For once the " at the beginning and end of some lines and as well the double "" where only one " should be. Probably some opening-error, I got it when I tried opening with excel. Should be fixed now however.

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -2936,31 +2936,31 @@ GetHideShow_Hide;Hide;;Ausblenden;;;;;;;;;;;x
...
 DEATH_PURGED;vanished in distant lands;;verschwand in fernen L�ndern.;;;;;;;;;;;x
 is_small_humanoid_tooltip;Is small humanoid;;Ist ein kleiner Humanoide;;;;;;;;;;;x
-<<<<<<< Updated upstream
+<<<<<<< Updated upstream;;;;;;;;;;;;;;
 desc_job_explorer;�YExplorer�! collects knowledge about Azeroth and opens up new horizons for you;�YExplorer�! Rassemble des connaissances sur Azeroth et ouvre de nouveaux horizons;Der �YErkunder�! sammelt Wissen �ber Azeroth und er�ffnet Euch neue Horizonte.;;;;;;;;;;;x
-=======
+Err:510;;;;;;;;;;;;;;
 desc_job_explorer;�YExplorer�! collects knowledge about Azeroth and opens up new horizons for you;;Der �YErkunder�! sammelt Wissen �ber Azeroth und er�ffnet Euch neue Horizonte.;;;;;;;;;;;x
->>>>>>> Stashed changes
+>>>>>>> Stashed changes;;;;;;;;;;;;;;
```
Comment (MemeWizard) [inline localisation/00_zuwc_united_localisation.csv]: I think something went wrong here

**Pull request merged**